resources/qml/TimelineRow.qml

```diff
@@ -85,6 +85,25 @@ Item {
             width: 16
         }
 
+        ImageButton {
+            id: editButton
+
+            visible: (Settings.buttonsInTimeline && model.isEditable) || model.isEdited
```

Comment:
If I'm reading this correctly, you're overloading this button to be for messages that are already edited as well as for messages that can be edited.  If this is the case, it seems like it could be confusing to users.  Also, what happens if someone tries to edit the same message twice?  the button will say 'Edited' when the user may want to edit again and it may lead them to think they cannot edit the message again despite a button being there that allows them to edit in other scenarios.

Reply:
Well, I don't really have a good idea on how to solve that. So I just left it like that for now and will later move the edit button into a dedicated action button bar, that shows on hover, while only status icons like read, edited and timestamp are shown by default?